### DISCUSSION TITLE
What icon size should I aim to when choosing icon for my audio devices?

### DISCUSSION THREAD

**Discussion by bigplayer-ai** [Belphemur/SoundSwitch#1108] [FAQ]
What icon size should I aim to when choosing icon for my audio devices? right now it shows very little icon on banners when switching audios.

**Comment by Belphemur** [comment]
The size that are going to matter for SoundSwitch are:
* 8x8 (for systray)
* 16x16 (for banner)
Also it needs to be an ico file as required by Windows.
